app/views/data/realtime/realtime.jade

```diff
@@ -16,7 +16,7 @@
     limitations under the License.
  
 link(rel="stylesheet", href="css/realtime.css")
-.container-fluid.height100(ng-controller="SessionCtrl" role="tabpanel" style="position:relative")
+.container-fluid.height100(ng-controller="ActivityCtrl" role="tabpanel" style="position:relative")
```

Comment:
Fixed in: https://github.com/e-ucm/rage-analytics-frontend/pull/90/commits/0edb046efc1121fa233acb2cd3d9afd7dd93ff4f